policy-controller/src/main.rs

```diff
@@ -10,10 +10,17 @@ use std::net::SocketAddr;
 use structopt::StructOpt;
 use tokio::{sync::watch, time};
 use tracing::{debug, info, info_span, instrument, Instrument};
+use tracing_subscriber::{fmt::format, prelude::*, reload, EnvFilter};
 
 #[derive(Debug, StructOpt)]
 #[structopt(name = "policy", about = "A policy resource prototype")]
 struct Args {
+    #[structopt(parse(from_str = LogLevel::new), long, default_value = "policy=info,warn")]
```

Comment:
In the destination template we pass in a default of `linkerd=info,warn`, however the default argument here is `policy=info,warn`. Is the default value here more for local development?

Reply:
good catch :)

```suggestion
    #[structopt(parse(from_str = LogLevel::new), long, default_value = "linkerd=info,warn")]
```